velox/core/QueryConfig.h

```diff
@@ -357,6 +357,43 @@ class QueryConfig {
   /// Empty string if only want to trace the query metadata.
   static constexpr const char* kQueryTraceNodeIds = "query_trace_node_ids";
 
+  /// Disable optimization in expression evaluation to peel common dictionary
+  /// layer from inputs.
+  static constexpr const char* kDebugDisableExpressionWithPeeling =
+      "debug_disable_expression_with_peeling";
```

Comment:
Would you send a PR to update contributing guidelines / coding style to describe when to use "debug_xxx" configs?

Reply:
@mbasmanova Created a seperate PR that I will merge after this since it references one of the configs added here: https://github.com/facebookincubator/velox/pull/10934